lib/wor/paginate/paginate.rb

```diff
@@ -0,0 +1,39 @@
+module Wor
+  module Paginate
+    ADAPTERS = [Adapters::KaminariAlreadyPaginated,
```

Comment:
Why not 

    ADAPTERS = [
      Adapters::KaminariAlreadyPaginated,
      Adapters::WillPaginateAlreadyPaginated,
      Adapters::WillPaginate,
      Adapters::Kaminari,
      Adapters::ActiveRecord,
      Adapters::Enumerable
    ].freeze